InventoryInStorePickupFrontend/view/frontend/layout/checkout_onepage_success.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceBlock name="checkout.success" class="Magento\InventoryInStorePickupFrontend\Block\Checkout\Onepage\Success" template="Magento_InventoryInStorePickupFrontend::success.phtml"/>
```

Comment:
Override template only, the block class should remain the same

Reply:
Done.